### PR TITLE
Permission write i workflows

### PR DESCRIPTION
**Pull request by klaramargrethehelgemo**
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] We need to implement grafana analytics

### CODE REVIEW

File: .github/workflows/build-deploy-feature-branch-q1.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ env:
   IMAGE_TAG: ${{ github.sha }}
   IMAGE: ghcr.io/${{ github.repository }}/veilarbportefolje
   PRINT_PAYLOAD: true
+permissions:
+  packages: write
```
Comment (OdaDahlen) [inline .github/workflows/build-deploy-feature-branch-q1.yaml:10]: Kan du raskt forklare hvorfor dette trengs? :)

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -4,6 +4,9 @@ env:
   IMAGE_TAG: ${{ github.sha }}
   IMAGE: ghcr.io/${{ github.repository }}/veilarbportefolje
   PRINT_PAYLOAD: true
+permissions:
+  packages: write
+
```
Comment (OdaDahlen) [inline .github/workflows/main.yml:9]: Kanskje du ikke vil ha den ekstra linja her? :)

File: .github/workflows/build-deploy-feature-branch-q1.yaml (hunk 1 above)
Comment (dragutin-nav) [reply .github/workflows/build-deploy-feature-branch-q1.yaml:10]: https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#about-scopes-and-permissions-for-package-registries

Comment (OdaDahlen) [reply .github/workflows/build-deploy-feature-branch-q1.yaml:10]: Fant også denne posten her: https://nav-it.slack.com/archives/D03UK0VHAF5/p1693825855373549

File: .github/workflows/main.yml (hunk 2 above)
Comment (klaramargrethehelgemo) [reply .github/workflows/main.yml]: Fjerna blanke linjer (en i hver workflow)

**Pull request merged**